### PR TITLE
Tweaked ElytronIntegrationChecker and ConfigChecker

### PR DESCRIPTION
**Pull request by honza-kasik**
* Added human like entering method
* Removed unnecessary code from ElytronIntegrationChecker, also now using documented public API
* Add possibility to choose input method in ConfigChecker

### CODE REVIEW

File: common/src/main/java/org/jboss/hal/testsuite/util/ConfigChecker.java [hunk 1]
```diff
@@ -154,10 +167,20 @@ public Builder configFragment(ConfigFragment config) {
...
          */
         public Builder edit(InputType inputType, String identifier, Object attrValue) {
+            return edit(inputType, identifier, attrValue, InputMethod.MACHINE);
+        }
+
+        /**
+         * use this method only if you need to set explicit {@link InputMethod}. Set field identified by
+         * <b>{@code identifier}</b> to be edited with <b>{@code attrValue}</b>. Can be called multiple times. The
+         * actual edit will be performed as soon as client calls the {@link #andSave()} </b>
+         * @param inputMethod Input method to be used. For example InputMethod.HUMAN to edit form like human.
```
Comment (zebrik) [inline common/src/main/java/org/jboss/hal/testsuite/util/ConfigChecker.java]: I would suggest to extend comment to emphasise that using this method is meant just as a workaround to observed weird behaviour of some input fields and should be used only in cases when standard `edit(InputType inputType, String identifier, Object attrValue)` method is not working properly.

File: common/src/main/java/org/jboss/hal/testsuite/fragment/formeditor/Editor.java [hunk 2]
```diff
@@ -51,6 +53,26 @@ public void text(String identifier, String value) {
     }
 
     /**
+     * Enters text as humanly as possible. E.g. makes pauses between characters and utilizes keyboard shortcuts.
```
Comment (zebrik) [inline common/src/main/java/org/jboss/hal/testsuite/fragment/formeditor/Editor.java]: Please add note that this method should be used only as a workaround in cases when `text(String, String)` is not working properly.

**Review by zebrik — COMMENTED** [review]
Just suggestions related to comments.

**Comment by honza-kasik** [comment]
Thanks! I improved comments and this is ready to be merged now!

**Comment by honza-kasik** [comment]
I need to build upon this, so I merged this myself, since errors were only in comments.

**Pull request merged**